features/media-capabilities.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Media Capabilities: Decoding
```

Comment:
how is that different from the previous one ?

Reply:
Oops, I ended up with a dupe! I've deleted it, and removed the character that I think was making travis fail

Reply:
yeah, `: ` is special in YAML, so using it inside a string requires quoting it.

Reply:
Good to know, thanks! Looks like we can merge this now then.